core/app/models/workarea/data_file/csv.rb

```diff
@@ -48,12 +48,22 @@ def export!
 
       def assign_attributes(model, attrs)
         model.fields.each do |name, metadata|
+          next if name == 'updated_at'
+
           value = CsvFields.deserialize_from(attrs, field: metadata, model: model)
-          model.send("#{name}=", value) if value.present?
+          changed = if value.is_a?(Time)
```

Comment:
Do you need this if we're just skipping `updated_at`?

Reply:
This fixes a bug with `created_at` always causing the record to update even though nothing changed. The CSV imports had an underlying bug wherein any records that are imported are touched, even though they don't actually get updated.

Reply:
I'm confused - below in the json version, you're specifically writing code to touch everything. Why can't we just let the oob Mongoid behavior do it's thing?

Reply:
I tried that first and it didn't work. When adding the tests for what happens in an embedded record, I was not seeing updated_at changing.